### ISSUE TITLE
Dependabot tried to update this pull request, but something went wrong. 

### ISSUE THREAD

**Issue by vorburger** [dependabot/dependabot-core#1969]
E.g. https://github.com/vorburger/fineract/pull/39 (and https://github.com/vorburger/fineract/pull/36 and presumably everything else on https://github.com/vorburger/fineract/pulls) fails with: _Dependabot tried to update this pull request, but something went wrong. We're looking into it, but in the meantime you can retry the update by commenting @dependabot rebase._ (which doesn't help)

It's possible (likely) that at least some of the proposed upgrades meanwhile happened in that project (independent of Dependabot PRs), but I have a vague memory of having seen it just close anything it sees is no longer required, instead of choking on it?

**Package manage/ecosystem**

Java / Gradle

@feelepxyz

**Comment by manuerumx** [comment]
This happens to me in a repository where the merge strategy only allows **Squash**.

**Comment by jeffwidman** [comment]
This error typically indicates something internal to the Dependabot infrastructure or an error in a native helper and not our ruby code went sideways.

However, it's been 2+ years since this issue was filed and a _lot_ has changed in our code / internal infra, so I'm going to close as not possible to reproduce.

If you're still seeing issues, please open a new ticket with more details including the job logs if possible and we'll try to be a bit quicker to help debug. 😃

**Comment by abelbraaksma** [comment]
FWIW, I am seeing this error many times a day. trying `@dependabot rebase` or `@dependabot recreate` _sometimes_ helps, but typically only after about 5 times or so.
